index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="http://">Option 1</a></li>
+                <li><a href="http://">Option 2</a></li>
+                <li><a href="http://">Option 3</a></li>
+            </ul>
+        </nav>
+    </header>
+    <div id="container">
+        <div id="sidebar">
+            <img src="./images/animals.jpg" height="50px" alt="Logo"><label>Select Video</label>
+            <ul>
+                <li><a href="http://"><img src="./images/bird.jpg" alt="Animals" width="30px"
+                            height="30px"></a><Label>   Birds</Label></li>
+            </ul>
+            <ul>
+                <li><a href="http://"></a><img src="./images/panda.jpg" alt="Animals" width="30px"
+                        height="30px"></a><Label>   Panda</Label></li>
+            </ul>
+            <ul>
+                <li><a href="http://"></a><img src="./images/squirrel.jpg" alt="Animals" width="30px"
+                        height="30px"></a><Label>   Squirrel</Label></li>
+            </ul>
+
+        </div>
+        <div id="content">
+            <section>
+                <h1>Pre-enrollment Video</h1>
+                <iframe src="https://www.youtube.com/embed/2IEvEXxQ2og" title="YouTube video player" frameborder="10"
+                    width=600px height=500px
+                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                    allowfullscreen></iframe>
+                    <h1>Amaizing nature</h1>
+                    <p>Video about the different bio diversity biomas</p>
+                    <label>number of views: (100)</label>
+            </section>
+        </div>
```

Comment:
In this portion of code, the use of the tag `div` is not necessary. Kindly consider removing it and use only the `section` to give this tag the CSS `class` attribute required.